Mapcycle update 2023

bunniewormy opened the pull request:
## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

previous mapcycle size: 67, now: 70

MODIFIED MAPS (17):
- bunnie_Elric - big revisions: changed entire mid area, reduced size, changed flag positions (were too easy to defend)
- JTG_FirstBadLands - added 2nd flag, made symmetrical, added gold near spawn (previously was on mid only, which was quite - stupid). Changed bedrock a bit
- Biurza_Shimamura - removed water, lowered sky height, lowered mid height, minor bedrock changes
- Biurza_My_Hikoboshi - changed mid so that controlling mid isn't so strong, reduced sky height, added some more stone 
- Ferrezinhre_Multi_Level_Crossing - removed top island, increased mid size. should be a bit harder to defend now.
- Asu_DeadPlateau - increased sky height, lowered height of mid (felt a bit cramped - you could make sky-high tower on mid really quickly once you got control of it); added a way for knights to get on top of the plateau at game start (if only one team had a builder that actually decides to go forward at game start, they pretty much always got mid). Minor bedrock modifications
- Asu_Kelsier - lowered sky height (often the team which made a higher tower than the other had a huge advantage over the other one due to the high sky & short mid area which then resulted in snowballing); moved 1 flag a bit, removed some anti-rat bedrock
- Coroz_floating_rifts - increased sky height and width (again, cramped); softened the anti-rat bedrock a bit (was way too harsh with only one narrow dirt passage into the island, so most of the map was usually played over the top, making the sky-island element not very interesting - the bedrock is still harsh, but not as limiting as earlier). Added an Elantrisque possibility for a bottom path under the mid islands. Added another flag for more interesting matches.
- Cleaf_Trumble - lowered sky height (was too high so mid towers were extremely strong), added another flag (games ended quite fast on it, since the only flag was pretty close to the red zone as well)
- petgreendino_skyson - increased mid size a bit to give some more breathing room. not gonna increase the area between tent & flags because that's an important part of the map's identity; it's both easy to defend and easy to attack (once you control mid), which results in quite intense matches
- petgreendin_bisonbisoff - matches ended too quickly on here, so added another flag, increased mid size (1 more bison for fun), lowered sky height, added small dirt island above flags to make defending against mid easier (its easy to attack from mid due to height advantage). removed some harsh anti-rat bedrock
- Punk123_Rally - added anti-rat bedrock, was way too ratty. changed some bedrock in upper mid too
- 4zK_Rorschach - the map is quite unique (cave map), but (as most of the maps i've mentioned at this point), it feels cramped. increased size, new mid. new version is probably very different but should still have the same spirit of the original
- GuyDeLu_Brokenland_v2 - reworked ass bedrock (stalemate extravaganza), moved tent a bit back
- Fellere_FourRivers - it often wins votes so it seems like people do like massive maps, so I think it's okay if we leave one big map in the cycle. edited it to be a bit more playable - less trees, better material spread, slight size reduction
- ffsff_PlatformFlats - shape of the team terrain was extremely sus and made it very easy to defend; a bit more "standard" now, but mid is pretty much the same.
- Didgedy_TwinTowers - author requested modifications

REMOVED MAPS (19):
- 8x_PorkBuns - it feels a bit too tight & narrow for pub matches
- bunnie_OkabeRintaro - pretty stale, been in the mapcycle for very long now, and is not very original; mostly flat
- bunnie_MapSpeedrun1 - shape allows for too easy defense, in general feels very weird and uncomfortable to play, especially for knights
- bunnie_ten-maly-karakan - uncomfortable for knights; due to the islands, bottom lane isn't played very often, very builder focused
- bunnie_CobblinGoblin - another multi-lane map where the top lane is way too strong, not very fit for pubs
- Mazey_Junctions - another map simply unsuited for pubs. a good builder can constantly bridge over from top and win the games very quickly; frustrating to play against as well
- Hugobh_Grottolands - again, shape issue - once the enemy takes the island before the flag, you've pretty much lost
- JTG_Aqueduct - control of mid is too strong
- Punk123_BlockCity - very ratty, very wide, flags hard to defend, low sky and not much else going on
- Punk123_PrincessCity2 - too small
- Punk123_Hillo - builder focused, very small mid area
- Ferrezinhre_Voltaric - too small
- Ferrezinhre_Reaving_Trauma - it's a less fun version of Rost. Water map - ok, but sky is way too low, it's too wide, the bedrock is unfun to play around.
- Ferrezinhre_Alchemy - extremely builder-focused, games end fast, quite hard to defend
- Ferrezinhre_Echo_No_2_Replica - was a weird map in general. hard to defend, mid control is strong and hard to get back, uncomfortable bedrock
- Potatobird_CubeCanyon - low sky, small, tight and narrow extremely uncomfortable for knights; mid is too easy to control (and too hard to get back) due to the height advantage and the bedrock
- Potatobird_SharkRomance_v1 - another rat extravaganza map that is unfun for anyone that doesn't love digging dirt and spamming doors
- SJD360_Stampede - too small, ratty, upper mid is too strong
- 4zK_Stubs - too small

ADDED MAPS (22):
- CerberosRift - adding back modified, wider with new mid and bedrock changes
- NewTortuga - adding back unmodified
- bunnie_Luthadel - added back, unmodified
- bunnie_Stoneward - adding in, flattish sky map
- bunnie_SkyAlchemy - extremely modified version of Alchemy; sky map with an Elantris vibe
- bunnie_FlatPleasure - adding in, nothing special, flattish standard map
- bunnie_TotalFlat - adding in, totally flat map because why not
- Biurza_Adapt_Overcome - adding in unmodified, mid-dirt-wall gimmick, let's try
- Biurza_Big_Brain - adding in unmodified, pit map
- FG_FourLakes - adding in unmodified, has high rat possibilities but the bedrock is quite brutal
- FG_Unearthed - adding in unmodified, might be quite big but let's give it a try
- FG_99 - adding in modified, made symmetrical & slight modifications. Standard flat map, shouldn't be controversial
- Ferrezinhre_Aojiroku - adding in modified, slightly increased mid width and area between flags and tent + less anti-rat bedrock. Allows for interesting multi-lane gameplay, might be fun.
- Ferrezinhre_Aurochs - adding in modified, standardish map with fun mid, maybe slightly ratty
- Ferrezinhre_Echo_Dissociation - adding in unmodified, Glitchesque map, might be fun
- Ferrezinhre_Golden_Melancholy - adding in modified (higher sky), another sky map
- Ferrezinhre_Highlands - adding back unmodified, was a long time since its been in cycle
- Ferrezinhre_Submissive_Transfiguration - adding in unmodified, quite yuge but can be interesting
- Ferrezinhre_Sunrinhelm - adding in modified, added some dirt islands as support for the bridge & added a flag. Very experimental but worth trying out.
- Ferrezinhre_Voice_Universal_Nemesis - adding in modified, moved flags down to make it harder to defend
- Ferrezinhre_We_will_kiss_in_the_moonlight - adding in umodified, gimmick map but might be more fun than a lot of the maps in.
- Gix_Devastated_Lake - adding in modified, changed flag position & minor modifications. Might be hard to attack but let's try it


mostly focused on getting rid of maps that just simply end too quickly because of bad shape or maps that are too builder-focused. it's not really fun when you instantly are at a huge disadvantage because the enemy team took some territory in the first few minutes of the game.
(you can check map duration on kagstats as well - got rid of everything with < 20 min avg match duration)

modified maps that had minor issues but were still pretty good overall

added some maps, ranging from pretty standard ones to sky maps and pretty unique ones

mapcycle should feel pretty fresh now. replaced around 30% of the maps and modified almost 25%

ferrezinhre commented:
Few questions and request if possible

Ferrezinhre_Aurochs - why mid was butchered and bedrock lifted? I don't see them as any neccessary changes, you removed possibility of flooding bellow chambers, that you don't like some aspects dosen't mean they require change.

Ferrezinhre_SkyAlchemy - never made this one, that it have "parts" of mine map dosen't mean is mine, keep your amalgamations for yourself. If you don't understand, remove my username from map that wasn't made by me

Now about JTG_FirstBadLands
Map was terrible yes, gold changes fine, little asymmetry on mid won't kill anyone.

AsuMagic commented on Maps/Official/CTF/Ferrezinhre_Aurochs.png:
Seems more ratty than average

bunniewormy replied:
it is more ratty than average, but i don't think it should be made less ratty. i think it's fine if there's a few rat-heavy maps in the cycle

+ top mid is still strong enough for ratting to not be the only way to win

AsuMagic commented on Maps/Official/CTF/Ferrezinhre_Sunrinhelm.png:
Maybe having one of the flags at the bottom would be more interesting? I'm not sure. The topmost flag seems relatively difficult to reach especially as the bridge is going to get shredded relatively quickly.

bunniewormy replied:
moved the flag to bottom passage

AsuMagic commented on Maps/Official/CTF/Punk123_Rally_v2.png:
This is added but not mentioned in the PR. Either way LGTM

bunniewormy replied:
fixed

bunniewormy commented on Maps/Official/CTF/Asu_Kelsier_v2.png:
lowered it a bit more

bunniewormy commented on Maps/Official/CTF/JTG_FirstBadLands_v2.png:
reduced the gold, 26 gold blocks (7 in team side, 12 in mid) now instead of 4
(original had 24 in middle)

AsuMagic commented on Maps/Official/CTF/4zK_Rorschach_v2.png:
The followup changes seem good. I generally agree with the new proportions especially on top, which hopefully should make top slightly harder to defend and more interesting to play, and hopefully the bottom plays a bit better as well.

bunniewormy commented on Maps/Official/CTF/Biurza_Big_Brain.png:
changed mid entirely

bunniewormy commented:
Changes:
Ferrezinhre_Sunrinhelm - removed top flag, added flag to the bottom passage
Biurza_Big_Brain - changed mid entirely and moved back flag a bit to make it less stalematey
Asu_Kelsier_v2 - made sky even lower
JTG_FirstBadLands_v2 - reduced the gold to 26 gold blocks (7 in team side, 12 in mid) instead of 40 (original had 24 in middle)
4zK_Rorschach - added v3 version, which is closer to the original. slightly bigger, has a stronger top mid & stronger bottom mid
